evaluation/docker-compose-isolated-services.yml

```diff
@@ -0,0 +1,58 @@
+# compose file for local dev with isolated backend and services container
+# (db migration is done in backend container)
```

Comment:
don't understand the naming - there is no backend service in this compose file? Lost track of all the different setups - overview/simplify or later?